src/inspector_agent.cc

```diff
@@ -304,7 +307,12 @@ class NodeInspectorClient : public V8InspectorClient {
         running_nested_loop_(false) {
     client_ = V8Inspector::create(env->isolate(), this);
     // TODO(bnoordhuis) Make name configurable from src/node.cc.
-    contextCreated(env->context(), GetHumanReadableProcessName());
+    ContextInfo info(
+        String::NewFromUtf8(env->isolate(),
+                            GetHumanReadableProcessName().c_str(),
+                            NewStringType::kNormal).ToLocalChecked());
```

Comment:
I would prefer if ContextInfo used `std::string` because then you don't have to worry whether it's safe to call into V8 from here and what the lifetime of the `Local<String>` is (and also because it's a bit awkward to create a JS string here, only to unpack it almost immediately again.)